node/node_cross_shard.go

```diff
@@ -49,7 +49,7 @@ func (node *Node) BroadcastCXReceipts(newBlock *types.Block, lastCommits []byte)
 		}
 		cxReceipts, err := node.Blockchain().ReadCXReceipts(uint32(i), newBlock.NumberU64(), newBlock.Hash(), false)
 		if err != nil || len(cxReceipts) == 0 {
-			utils.Logger().Warn().Err(err).Uint32("ToShardID", uint32(i)).Int("numCXReceipts", len(cxReceipts)).Msg("[BroadcastCXReceipts] No ReadCXReceipts found")
+			utils.Logger().Info().Err(err).Uint32("ToShardID", uint32(i)).Int("numCXReceipts", len(cxReceipts)).Msg("[BroadcastCXReceipts] No ReadCXReceipts found")
```

Comment:
why is this just info not error?

Reply:
Its flooding my logs with warnings constantly, and I have an alert system that triggers if it sees enough warnings.
From slack:
>>>[12:28 AM] Chris | Harmony: That probably should be info
[12:28 AM] Chris | Harmony: That happens whenever there's no cross shard transactions
[12:28 AM] Chris | Harmony: Which is probably pretty common
[12:28 AM] Chris | Harmony: Also for all 3 shards